### ISSUE TITLE
[Bug] [a11y] Onboarding UI - The selected enabled options are not recognized as activated

### ISSUE THREAD

**Issue by Diana-Rus** [mozilla-mobile/fenix#8581]
Pre-conditions:
- Have a fresh clean profile of the application
- Have "TalkBack" option enabled

## Steps to reproduce
1. Open app.
2. From the "Onboarding" screen, section "Protect yourself" tap on "Strict (recommended)".

### Expected behavior
The "Strict (recommended)" is active by default. The TalkBack app recognizes the option as activated.
### Actual behavior
The "Strict (recommended)" is active by default. The TalkBack app does not recognize the option as activated.
### Device information

* Android device: Google Pixel C (Android 8.0.0), Samsung Galaxy S8 (Android 9)
* Fenix version: Firefox Preview Beta 4.0.0-beta.1

Note: 
- The issue reproduces on the "Onboarding" screen for active options from sections:
                        - "Choose your theme" - "Light theme" (activated/enabled)
                        - "Take a position" - "Top" (activated/enabled) which is for toolbar
- I consider that the outcome for the activated options should be as it is now when going to with TalkBack enabled:
                        - "App Menu"/"Settings"/"Customize"/"Toolbar" section/ option "Top" selected(TalkBack - recognized it as "checked")

**Comment by asadotzler** [comment]
This is an accessibility P2. The feature is available and works but is difficult to use. This kind of bug prevents Fenix from being a delightful experience for users with disabilities.

**Comment by sflorean** [comment]
This issue is reproducible for all settings that have radio buttons/ switch options.

**Comment by ebalazs-sv** [comment]
This is reproducible on Firefox Preview v5.0-beta.1 with Motorola Moto G6 (Android 8).

**Comment by mcarare** [comment]
This issue is included in https://github.com/mozilla-mobile/fenix/issues/9722.
